canonical-web-and-design/maas-ui
# refactor(ui): refactor the network table to handle controllers

## Done

- Update the network table to handle controllers.
- Move the components to the base dir.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the network tab for a machine and check that the table appears unchanged.
- Check that you can add/edit interfaces (for a new, ready, allocated or broken machine).

## Fixes

Fixes: canonical-web-and-design/app-tribe#940.
